core/modules/czar/bin/startCzar.py

```diff
@@ -113,15 +103,19 @@ def main():
     log.debug("Configuration:\n%s", config.toString())
 
     if options.test == True:
+        # not working
         runParserTest()
         return
     elif options.testName:
+        # not working
         runNamedTest(options.testName)
         return
     elif options.sanityClient:
+        # not working
         client.runSanityClient()
         return
     elif options.makeIndex:
+        # not working
```

Comment:
These were probably working long time ago but not recently. I tried to run these tests before doing my changes and it all crashed. This stuff will disappear anyways and I only added comments there to see what pieces are still functional and need to be ported to C++.